### PR TITLE
completed: Replace Spritz finance on UI with dragons dispatch

### PR DESCRIPTION
**Pull request by CodeSnooker**

### CODE REVIEW

**Comment by CodeSnooker** [comment]
[Replace Spritz finance on UI with dragons dispatch](https://trello.com/c/ey1kKynu/488-replace-spritz-finance-on-ui-with-dragons-dispatch)

File: src/pages/SwapPage/BuySellWidget/DraginDipatchAdvertisement.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import { Box, Button, CircularProgress } from '@material-ui/core';
```
Comment (sameepsi) [inline src/pages/SwapPage/BuySellWidget/DraginDipatchAdvertisement.tsx:1]: Please fix the name of this file

**Review by sameepsi — CHANGES_REQUESTED** [review]

File: src/pages/SwapPage/BuySellWidget/DraginDipatchAdvertisement.tsx (hunk 1 above)
Comment (CodeSnooker) [reply src/pages/SwapPage/BuySellWidget/DraginDipatchAdvertisement.tsx]: done

File: src/pages/SwapPage/BuySellWidget/DragonDipatchAdvertisement.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import { Box, Button, CircularProgress } from '@material-ui/core';
```
Comment (sameepsi) [inline src/pages/SwapPage/BuySellWidget/DragonDipatchAdvertisement.tsx:1]: fix the name of this file

**Review by sameepsi — APPROVED** [review]

**Pull request merged**